docker/local-run/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pcic/pdp-base-minimal:1.0.0
+FROM pcic/pdp-base-minimal:pdp-python3
```

Comment:
Add a comment that this should be replaced by a release tag when that branch has been merged.